proto/buf/alpha/audit/v1alpha1/event.proto

```diff
@@ -80,11 +100,135 @@ message Event {
+  buf.alpha.registry.v1alpha1.OrganizationRole new_role = 4;
+}
+
+message PayloadOrganizationMemberRemoved {
+  // member_id is the id of the member removed from the organization.
+  string member_id = 1;
+  // member_name is the name of the member removed from the organization.
+  string member_name = 2;
+  // old_role is the old role of the member removed from the organization.
+  buf.alpha.registry.v1alpha1.OrganizationRole old_role = 3;
+}
+
+message PayloadRepositoryCreated {
+  // create_time is when the repository was created.
+  google.protobuf.Timestamp create_time = 1;
+}
+
+message PayloadRepositoryDeleted {}
+
+message PayloadRepositoryPushed {
+  // owner_id is the id of the owner of the repository.
+  string owner_id = 1;
+  // owner_name is the name of the owner of the repository.
+  string owner_name = 2;
+  // commit is the name of the created commit.
+  string commit = 3;
+  // tags are the tags included in the push. Optional.
+  repeated string tags = 4;
+  // draft_name is the name of the draft. Optional.
+  string draft_name = 5;
+}
+
+message PayloadRepositoryMemberAdded {
+  // member_id is the id of the member added to the repository.
+  string member_id = 1;
+  // member_name is the name of the member added to the repository.
+  string member_name = 2;
+  // new_role is the role granted to the member added to the repository.
+  buf.alpha.registry.v1alpha1.RepositoryRole new_role = 3;
+}
+
+message PayloadRepositoryMemberRoleChanged {
+  // member_id is the id of the member whose role was changed.
+  string member_id = 1;
+  // member_name is the name of the member whose role was changed.
+  string member_name = 2;
+  // old_role is the old role of the member whose role was changed.
+  buf.alpha.registry.v1alpha1.RepositoryRole old_role = 3;
+  // new_role is the new role of the member whose role was changed.
+  buf.alpha.registry.v1alpha1.RepositoryRole new_role = 4;
+}
+
+message PayloadRepositoryMemberRemoved {
+  // member_id is the id of the member removed from the repository.
+  string member_id = 1;
+  // member_name is the name of the member removed from the repository.
+  string member_name = 2;
+  // old_role is the old role of the member removed from the repository.
+  buf.alpha.registry.v1alpha1.RepositoryRole old_role = 3;
+}
+
+message PayloadPluginCreated {
+  // create_time is when the plugin was created.
+  google.protobuf.Timestamp create_time = 1;
+}
+
+message PayloadPluginDeleted {}
+
+message PayloadPluginPushed {
+  // owner_id is the id of the owner of the plugin.
+  string owner_id = 1;
+  // owner_name is the name of the owner of the plugin.
+  string owner_name = 2;
+}
+
+message PayloadUserRegistered {
+  // create_time is when the user was created.
+  google.protobuf.Timestamp create_time = 1;
+}
+
+message PayloadUserDeactivated {}
+
+message PayloadUserDeleted {}
+
+message PayloadUserLoggedIn {}
+
+message PayloadUserLoggedOut {}
```

Comment:
Not sure if we can have additional details here like an IP address? Not blocking, this could be figured out in that specific ticket.

Reply:
yeah, figured I'd leave it out of here and we could decide when implementing.